app/src/main/java/it/niedermann/nextcloud/deck/repository/SyncRepository.java

```diff
@@ -183,7 +182,7 @@ public void onError(Throwable throwable) {
                             syncHelper.setResponseCallback(callback);
 
                             try {
-                                syncHelper.doSyncFor(new BoardDataProvider(progress$));
+                                syncHelper.doSyncFor(new BoardDataProvider(progress$, false), false);
```

Comment:
This is effectively called with hard coded `false`s by:

- Background synchronization
- Importing any account
- Switching account
- Regaining network connection
- Getting a Push Notification that is not related to a specific card

Can you please review the list to ensure that this is necessary / correct for all of the mentioned use cases or should we parametrize this and, just for example, keep parallel sync for the background synchronization? I don't know it, but I stumbled upon it and wanted to ask.

Reply:
As stated above: none of this is mandatory, it was created in the process of finding the problems with the sync. You want to get rid of it? Ok! I was really desperate and tried around. Something seems to have done something, now we're far more stable. I can't really tell if this is part of the solution, since my mind blew up with all the concurrency going on here. If you have a strong opinion on this: tell me and/or just implement it.